x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -81,38 +83,26 @@ export const ConditionEntry = memo<ConditionEntryProps>(
           inputDisplay: CONDITION_FIELD_TITLE[ConditionEntryField.PATH],
           value: ConditionEntryField.PATH,
         },
+        ...((os !== OperatingSystem.WINDOWS && []) || [
+          {
+            inputDisplay: CONDITION_FIELD_TITLE[ConditionEntryField.SIGNER],
+            value: ConditionEntryField.SIGNER,
+          },
+        ]),
       ];
-    }, []);
+    }, [os]);
 
     const handleValueUpdate = useCallback<ChangeEventHandler<HTMLInputElement>>(
-      (ev) => {
-        onChange(
-          {
-            ...entry,
-            value: ev.target.value,
-          },
-          entry
-        );
-      },
+      (ev) => onChange({ ...entry, value: ev.target.value }, entry),
```

Comment:
so clean
![rsz_94f](https://user-images.githubusercontent.com/315796/100750591-40400780-33b4-11eb-8fdc-9ae99444bd0f.png)

Reply:
Why are you only showing it now to me that it's possible to post meme's in the comments to PR!!!